### PR TITLE
14-2-en

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: 12-01-rerere/12-01-rerere.en.sbv [hunk 1]
```diff
@@ -8,22 +8,22 @@ have to be resolved once again. For example
...
 0:00:09.045,0:00:12.015
-Resolve conflicts from them. And here happens that we have to --abort the rebase.
+Resolve conflicts from them. And here happens that we have to '--abort' the rebase.
 
 0:00:12.015,0:00:14.705
- Thus when we rebase again
+Thus when we rebase again
 
 0:00:14.705,0:00:17.445
 will emerge the same conflicts.
```
Comment (yaroslavya) [inline 12-01-rerere/12-01-rerere.en.sbv:17]: "Later": word order, like: same conflicts will emerge.
Here and further I just state the things we'll have to fix later. For now its ok

File: 12-01-rerere/12-01-rerere.en.sbv [hunk 2]
```diff
@@ -8,22 +8,22 @@ have to be resolved once again. For example
...
 
 0:00:14.705,0:00:17.445
 will emerge the same conflicts. 
 
 0:00:17.445,0:00:20.485
-Or if we've just merge but need to revert it. 
+Or if we've just merged but need to revert it. 
 
 0:00:20.485,0:00:22.935
 To make some  minor needed changes.
```
Comment (yaroslavya) [inline 12-01-rerere/12-01-rerere.en.sbv:23]: "Later, order" some minor changes needed.

File: 12-01-rerere/12-01-rerere.en.sbv [hunk 3]
```diff
@@ -44,7 +44,7 @@ Then presumably you'll meet same conflicts.
 You can easily find yourself situations in development process where you have to resolve conflicts in repeat, better than I'll list them for you.
```
Comment (yaroslavya) [inline 12-01-rerere/12-01-rerere.en.sbv:44]: "Later" rephrase and, maybe, split into several sentences.

File: 12-01-rerere/12-01-rerere.en.sbv [hunk 4]
```diff
@@ -53,25 +53,25 @@ Happily ever Git is able to do it automatically.
 With help of mechanism called 'rerere'.
```
Comment (yaroslavya) [inline 12-01-rerere/12-01-rerere.en.sbv:53]: "Later, order, rephrase"

File: 12-01-rerere/12-01-rerere.en.sbv [hunk 5]
```diff
@@ -53,25 +53,25 @@ Happily ever Git is able to do it automatically.
...
- Shortened from REuse, REcorded, REsolution.
+Shortened from REuse, REcorded, REsolution.
 
 0:01:00.660,0:01:04.995
-When you enable this mechanism. Git will start to remember how was conflicts were resolved.
+When you enable this mechanism. Git will start to remember how conflicts were resolved.
 
 0:01:04.995,0:01:07.565
-So even this conflict pops up one more time
+So even same conflict pops up one more time
```
Comment (yaroslavya) [inline 12-01-rerere/12-01-rerere.en.sbv:62]: "later, rephrase"

File: 12-01-rerere/12-01-rerere.en.sbv [hunk 6]
```diff
@@ -86,7 +86,7 @@ which appears only If 'rerere' is working. And means
 that 'rerere' mechanism remembered current state for index.html
```
Comment (yaroslavya) [inline 12-01-rerere/12-01-rerere.en.sbv:86]: "later, rephrase"

File: 12-01-rerere/12-01-rerere.en.sbv [hunk 7]
```diff
@@ -160,7 +160,7 @@ Means 'rerere' applies to file conflict resolution it remembers but didn't add i
 
 0:02:51.180,0:02:56.840
 Option you can easily add by 
-git config rerere.autoupdate true
+'git config rerere.autoupdate true'
 
 0:02:57.260,0:02:59.280
 Special setting of config for 'rerere'
```
Comment (yaroslavya) [inline 12-01-rerere/12-01-rerere.en.sbv:166]: "later, rephrase", special "rerere" config setting

File: 12-01-rerere/12-01-rerere.en.sbv [hunk 8]
```diff
@@ -418,7 +418,7 @@ Finally I will mention that script 'rerere train'
 is written in shell language (sh)
 
 0:07:21.440,0:07:25.480
-Thas why on Windows it will work using git bash
+Thas why on Windows it will work using Git bash
```
Comment (yaroslavya) [inline 12-01-rerere/12-01-rerere.en.sbv:421]: "later, typo" Thas->That's

File: 14-02-date-output/14-2-date-output-formatting-en.sbv [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+which means date of the commit in default format
+
+0:00:20.840,0:00:24.280
+then I will place commit header '%s'
+
+0:00:24.520,0:00:27.240
+to make the date shorter
+
+0:00:27.240,0:00:28.980
+let's configure it's format
```
Comment (yaroslavya) [inline 14-02-date-output/14-2-date-output-formatting-en.sbv:33]: "later, typo" it's->its

File: 14-02-date-output/14-2-date-output-formatting-en.sbv [hunk 10]
```diff
@@ -0,0 +1,276 @@
...
+
+0:00:29.140,0:00:31.740
+'git log' command has for this purpose flag '--date'
```
Comment (yaroslavya) [inline 14-02-date-output/14-2-date-output-formatting-en.sbv:36]: "later, rephrase", e.g. 'git log' has a '--date' flag for exactly this

**Review by yaroslavya — CHANGES_REQUESTED** [review]
Good job! Needs some polishing imo. Made some comments where it's most obvious. @shydictator  can you, please, change it a little so I can merge it in? Let me know if you need more details on what/how to change.

**Comment by iliakan** [comment]
@yaroslavya maybe could be easier to merge and fix?

**Comment by yaroslavya** [comment]
Done, will fix this weekend.

**Pull request merged**